app.js

```diff
@@ -13,13 +14,34 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
+hbs.registerPartials(__dirname + '/views/partials');
 // ...
 
 // Add the route handlers here:
-
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
+});
+
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers.hbs', { beersFromApi });
```

Comment:
Please remove your `console.log();` after you're finished confirming that everything works.